msteams-platform/apps-in-teams-meetings/meeting-app-extensibility.md

```diff
@@ -0,0 +1,158 @@
+---
+title: Meeting app extensibility
+author: laujan
+description: Understand the meeting app extensibility 
+ms.topic: conceptual
+---
+
+# Meeting app extensibility
+
+Teams’ meeting app extensibility is based on the following concepts:
+
+✔ Meeting lifecycle has stages such as before, during, and after meeting time frame.  
```

Comment:
drop "time frame" 
Instead - Meeting lifecycle has stages such as before, during, and after the meeting